src/css/map-view.css

```diff
@@ -1472,34 +1473,109 @@ other class: .ui-slider-range */
   flex-direction: column;
+  margin-top: 1rem;
+  padding: .25rem .5rem;
   row-gap: .25rem;
+  color: black;
 
-  .viewfinder-zoom-presets__preset {
+  .viewfinder-zoom-preset__preset {
     border-radius: var(--map-border-radius-small);
     border: 1px solid var(--map-col-section-divider, var(--map-col-bkg-lightest__deprecate));
     cursor: pointer;
-    padding: .5rem;
+    display: flex;
+    flex-direction: column;
+    padding: .5rem 2rem;
+    row-gap: .5rem;
 
     &:hover {
-      background-color: var(--map-col-bkg-lightest);
+      background-color: var(--map-col-content-bkg-highlight, #15324e);
+    }
+
+    .viewfinder-zoom-preset__title {
+      color: var(--portal-col-buttons-text, white);
+      font-weight: bold;
+    }
+
+    .viewfinder-zoom-preset__description {
+      color: var(--portal-col-text-label, white);
+    }
+
+    .viewfinder-zoom-preset__layers {
+      column-gap: 4px;
+      display: flex;
+      flex-wrap: wrap;
+      overflow: hidden;
+      row-gap: 4px; 
+    }
+
+    .viewfinder-zoom-preset__layer {
+      align-items: center;
+      background: var(--portal-col-layer-label-bkg, var(--map-col-bkg-lighter__deprecate));
+      border-radius: 4px;
+      display: flex;
+      height: 24px;
+      justify-content: center;
+      max-width: 200px;
+      overflow: hidden;
+      padding: 0 16px 0 8px;
+
+      .viewfinder-zoom-preset__layer-content {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        text-wrap: nowrap;
+        color: var(--portal-col-text-muted, white);
+        font-size: .75rem;
+        max-width: 100px;
+        
+        i {
+          padding-right: 4px;
+          color: var(--portal-col-buttons-text, white);
+        }
+      }
     }
   }
 
   .viewfinder-zoom-preset.viewfinder-zoom-preset--active .viewfinder-zoom-preset__preset {
     background-color: var(--portal-col-buttons-bkg-highlight, var(--map-col-bkg-lighter__deprecate));
-    color: white;
 
+    .viewfinder-zoom-preset__title,
     .viewfinder-zoom-preset__description {
       color: white;
     }
+
+    .viewfinder-zoom-preset__layer {
+      background: var(--portal-col-layer-label-bkg-active, #15324e);
+
+      .viewfinder-zoom-preset__layer-content {
+        color: var(--portal-col-buttons-bkg-hover, white);
+
+        i {
+          color: var(--portal-col-layer-label-icon-bkg, white);
+        }
+      }
+    }
   }
 }
 
 /** START - Expansion panel View. */
 .expansion-panel {
-  border: solid var(--map-col-item-divider, var(--map-col-bkg-lightest__deprecate));
-  border-width: 1px 0;
-  padding: .25rem .5rem;
+  box-shadow: 0 1px var(--map-col-item-divider, var(--map-col-bkg-lightest__deprecate)) inset;
+
+  .expansion-panel__toggle {
+    display: grid;
+    align-items: center;
+    cursor: pointer;
+    font-size: 1rem;
+    grid-template-columns: min-content auto min-content;
+    padding: 0 1.5rem;
+    border: solid var(--map-col-item-divider, var(--map-col-bkg-lightest__deprecate));
```

Comment:
This matches the UX in layer categories. Layer categories should eventually use this shared component instead of its own solution. Which do you prefer?

Reply:
def reusing this one after it's done! but this is the extra bottom border I'm talking about that's not matching the mock:
![image](https://github.com/NCEAS/metacatui/assets/8570665/cf71e2e2-a5e6-456a-b452-a99899417616)

Reply:
Understood, but this is what it looks like in layer categories, should I follow the mocks or the current implementation that is live in prod?
![borderbottom](https://github.com/NCEAS/metacatui/assets/4664309/85aa985f-2a0d-45d5-9dce-e251ba628876)

Reply:
Synced with UX, no border bottom is the way to go

Reply:
OH!! okay I know what happened in the layers list- an issue I introduced when fixing Doug's css changes🤦🏻‍♀️